### DISCUSSION TITLE
What are the system requirements for setting up this program?

### DISCUSSION THREAD

**Discussion by RayBB** [d2phap/ImageGlass#671] [General] [answered]
## Questions

Hello,
I'm looking to get ImageGlass setup with Wine / PlayOnLinux and I'm trying to figure out the system requirements to make sure all the dependencies are installed.
So far all I see is dot net 4.7.1. Is there anything else required?

Thanks

**Comment by d2phap** [comment, marked as answer]
Hi @RayBB 
Yes the only requirement is .NET Framework 4.7.1 for Windows OS. I am not sure about running on Wine/PlayOnLinux environment on other platform.

**Comment by RayBB** [comment]
Thanks for the quick response!
Is it 32 bit compatible or only 64 bit?

**Comment by d2phap** [comment]
it's both compatible with 32 and 64 bit
